tests/VolleyManagement.UnitTests/Services/GameService/GameServiceTests.cs

```diff
@@ -1648,6 +1648,42 @@ public void Edit_GameValid_GameEdited()
             VerifyEditGame(game, Times.Once());
         }
 
+        /// <summary>
+        /// Test for Edit method. Game object in PlayOff contains valid data. 
+        /// No exceprtion returns and game is edited successfully
+        /// </summary>
+        [TestMethod]
+        public void Edit_GameTimeForPlayOffChanged_NoException()
```

Comment:
Your case is to wide. We need to make it more specific.

You've added condition stating HomeTeamId!=null. What does it mean from user perspective?

Reply:
@sdiachen It means that if home team in the game isn't known, we can't find out the winner of this game. And we can't pass the winner of this game to the next round too.

Reply:
Is it possible that I don't need to check if game time changes but only if the winner of the game is chosen correctly?

Reply:
Your case here is that you don't have all teams defined.

_EditTimeForPlannedGame_GameSaved
_EditTimeForUpcomingGameSlot_GameSaved